client/packages/common/src/intl/locales/en/app.json

```
@@ -55,6 +55,7 @@
   "error.integration-timeout-reached": "The central server took too long integrating pushed records. Please retry later.",
   "error.invalid-url": "Invalid url",
   "error.login": "Invalid username or password",
+  "error.login-support": "Please contact support at support@msupply.foundation if you are unable to login to your account",
```

our other 'contact support' messaging has this format: no need to say 'support at support@'
```suggestion
  "error.login-support": "Please contact support@msupply.foundation if you are unable to login to your account",
```